modules/apps/change-tracking/change-tracking-api/src/main/java/com/liferay/change/tracking/scheduler/PublishScheduler.java

```diff
@@ -22,6 +22,6 @@ public ScheduledPublishInfo getScheduledPublishInfo(
 	public void schedulePublish(long ctCollectionId, long userId, Date date)
 		throws PortalException;
 
-	public void unschedulePublish(long ctCollectionId) throws PortalException;
+	public void unschedulePublish(long ctCollectionId) throws Exception;
```

Comment:
You don't need to fix these exceptions manually. They should be updated when you run service builder.